utilities/functions.py

```diff
@@ -31,3 +31,10 @@ def rescale_js(
 def constrain_angle(angle: float) -> float:
     """Wrap an angle to the interval [-pi, pi]."""
     return math.atan2(math.sin(angle), math.cos(angle))
+
+
+def rotate_vector(x: float, y: float, angle: float) -> (float, float):
```

Comment:
We should probably use this function everywhere we do this too (we do this in the swerve code too).